components/TeamMember/TeamMember.test.tsx

```diff
@@ -0,0 +1,58 @@
+import { render, screen } from "@testing-library/react";
+import { TeamMember } from "components/TeamMember/TeamMember";
+
+const member = {
+  _id: "1",
+  name: "Miguevara",
+  surname: "Pantoja",
+  pronouns: "Él",
+  role: "Capitan",
+  contactLinks: [
+    { name: "Instagram", url: "https://instagram.com/john-doe" },
+    { name: "Linkedin", url: "https://linkedin.com/john-doe" },
+  ],
+};
+
+describe("TeamMember", () => {
+  it("renders the component", () => {
+    render(<TeamMember {...member} />);
+
+    const component = screen.getByTestId("team-member");
+    expect(component).toBeInTheDocument();
+  });
+
+  it("renders the profile image and placeholder", () => {
```

Comment:
Isn't this test failing for you? Are you in Node.js v16.8.0? :thinking:

Reply:
No, Husky would deny the commit if so... 
![image](https://github.com/Juguetear/juguetear-web/assets/28108101/eb83aa01-fb99-4b09-b84c-572fe146b8b0)

Reply:
@alexxuss1996 can you give some details on what Husky says?

Reply:
@TonyMckes Maybe Node 16 don't support Object.hasOwn(), it may be the reason, i

> @alexxuss1996 can you give some details on what Husky says?
> 
> I'm thinking it can be something related to Windows, I just need confirmation

I checked, it failed because Object.hasOwn() is supported from Node 16.9+

Reply:
Yes, that is why. But now I am more curious about Husky, since is not the first time has been reported by a contributor

To fix this issue we can change `if (Object.hasOwn(member, "image"))` for `if (member.image)`..
And then, import the interface `TeamMemberProps`, so.. `const member: TeamMemberProps = ...`

Reply:
@TonyMckes Done!